Dockerfile

```diff
@@ -16,15 +16,14 @@ RUN ./build-toolchain.sh
 
 # Strip executables
 RUN find ${N64_INST}/bin -type f | xargs strip
-RUN strip ${N64_INST}/libexec/gcc/mips64-elf/10.2.0/plugin/gengtype
-RUN strip ${N64_INST}/libexec/gcc/mips64-elf/10.2.0/liblto_plugin.so.0.0.0
-RUN strip ${N64_INST}/libexec/gcc/mips64-elf/10.2.0/liblto_plugin.so.0
-RUN strip ${N64_INST}/libexec/gcc/mips64-elf/10.2.0/lto-wrapper
-RUN strip ${N64_INST}/libexec/gcc/mips64-elf/10.2.0/collect2
-RUN strip ${N64_INST}/libexec/gcc/mips64-elf/10.2.0/cc1plus
-RUN strip ${N64_INST}/libexec/gcc/mips64-elf/10.2.0/cc1
-RUN strip ${N64_INST}/libexec/gcc/mips64-elf/10.2.0/install-tools/fixincl
-RUN strip ${N64_INST}/libexec/gcc/mips64-elf/10.2.0/lto1
+RUN strip ${N64_INST}/libexec/gcc/mips64-elf/11.2.0/plugin/gengtype
+RUN strip ${N64_INST}/libexec/gcc/mips64-elf/11.2.0/liblto_plugin.so
+RUN strip ${N64_INST}/libexec/gcc/mips64-elf/11.2.0/lto-wrapper
+RUN strip ${N64_INST}/libexec/gcc/mips64-elf/11.2.0/collect2
+RUN strip ${N64_INST}/libexec/gcc/mips64-elf/11.2.0/cc1plus
+RUN strip ${N64_INST}/libexec/gcc/mips64-elf/11.2.0/cc1
+RUN strip ${N64_INST}/libexec/gcc/mips64-elf/11.2.0/install-tools/fixincl
+RUN strip ${N64_INST}/libexec/gcc/mips64-elf/11.2.0/lto1
```

Comment:
I think you should be able to remove these if you switch to `install-strip`

Reply:
Done.